docs/release_notes/9_2.md

```diff
@@ -25,6 +25,30 @@ You can upgrade from Rocky Linux 9.x to Rocky Linux 9.2 by doing `sudo dnf -y up
 !!! Note
 
     Rocky Linux does not offer an upgrade path from any version of Rocky Linux 8. We recommend doing a fresh OS install to move to Rocky Linux 9.2.
+    
+!!! Bug "Upgrading with LVM devices may result in boot failure"
+
+    Due to changes in the lvm2 package between Rocky 9.0, 9.1, and 9.2, upon rebooting after upgrading your system may drop into recovery mode due to one or more LVM partitions not being found. We have opened [rhbz#2208039](https://bugzilla.redhat.com/show_bug.cgi?id=2208039) to track this regression.
+    
+    This bug affects both physical and virtual systems, though Libvirt(KVM), VMWare, as well as some other guests appear to not be affected.
```

Comment:
As tg has writte in the chat it might be better to not say `appear to not be affected` but `have been reported to not be affected`